src/ducks/apps/components/Sections/categories.js

```diff
@@ -57,14 +57,18 @@ export const addLabel = (cat, t) => ({
   label: t(`app_categories.${cat.value}`)
 })
 
-//
 /**
  * Returns unlabelised/unsorted categories from a list of apps
  * @param  {Array<App>}  apps
  * @param  {Boolean} includeAll - Whether to add an "All" option
  * @return {Array<CategoryOption>}
  */
-export const generateOptionsFromApps = (apps, includeAll = false) => {
+export const generateOptionsFromApps = (
+  apps,
+  includeAll = false,
+  _addLabel
```

Comment:
Maybe `includeAll` and `_addLabel` should be in an `options` object? They are optional, and it will be better for code reading when using this function: `generateOptionsFromApps(apps, { includeAll: true })`

Reply:
Done in https://github.com/cozy/cozy-store/pull/622/commits/5636fdb6108fedb74b5051ec0970fded22f4ef27